### DISCUSSION TITLE
How to create a manual order with a different product (variation) title?

### DISCUSSION THREAD

**Discussion by therealgilles** [woocommerce/woocommerce#36796] [General]
Is there a way to create a manual order with a product title override through the WP admin interface?
I can do it programmatically through hooks or the REST API, but I don't see a way to do with through the WP admin interface. Somehow the product title field is not editable when editing an order.

**Comment by jongore** [comment]
To create a manual order with a different product variation title in WooCommerce, you can follow these steps:

Go to the WordPress dashboard and navigate to WooCommerce > Orders.

Click on the "Add Order" button.

On the "Add Order" page, enter the customer information and order details.

In the "Order Items" section, click on the "Add item(s)" button.

In the "Product" field, start typing the name of the product or variation that you want to add to the order. You should see a list of suggestions that match your search terms.

Click on the product or variation that you want to add to the order.

In the "Item Name" field, you can enter a different title for the product variation that you're adding to the order.

Enter the quantity and other details for the product variation, such as price and tax.

Click on the "Add to Order" button to add the product variation to the order.

Continue adding other products or variations to the order as needed.

When you're done adding items to the order, click on the "Save Order" button to create the manual order with the different product variation title.

That's it! You've now created a manual order in WooCommerce with a different product variation title.

**Reply by therealgilles** [reply]
I do not see any "Item Name" field when I create a manual order.

**Reply by therealgilles** [reply]
It looks like it works if I use "_name" as meta field name, which is not obvious. Please let me know if I am somehow missing the real "Item Name" field.

**Reply by therealgilles** [reply]
I also get this PHP warning when using _name:
```PHP
is_internal_meta_key was called incorrectly. Generic add/update/get meta methods should not be used for internal meta data, including "_name".
Use getters and setters.
Backtrace: do_action('wp_ajax_woocommerce_save_order_items'), WP_Hook->do_action, WP_Hook->apply_filters, WC_AJAX::save_order_items, wc_save_order_items, WC_Data->update_meta_data, WC_Data->is_internal_meta_key, wc_doing_it_wrong. This message was added in version 3.2.0.
```
